aws/resource_aws_db_parameter_group_test.go

```diff
@@ -291,6 +291,15 @@ func TestAccAWSDBParameterGroup_basic(t *testing.T) {
 						"aws_db_parameter_group.bar", "arn", regexp.MustCompile(fmt.Sprintf("^arn:[^:]+:rds:[^:]+:\\d{12}:pg:%s", groupName))),
 				),
 			},
+			{
+				Config: testAccAWSDBParameterGroupConfig(groupName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSDBParameterGroupExists("aws_db_parameter_group.bar", &v),
+					testAccCheckAWSDBParameterGroupAttributes(&v, groupName),
+					testAccCheckAWSDBParameterNotUserDefined("aws_db_parameter_group.bar", "collation_connection"),
+					testAccCheckAWSDBParameterNotUserDefined("aws_db_parameter_group.bar", "collation_server"),
```

Comment:
To add some additional checking to this, you could try adding something like: 
```suggestion
					testAccCheckAWSDBParameterNotUserDefined("aws_db_parameter_group.bar", "collation_server"),
					testAccCheckNoResourceAttr("aws_db_parameter_group.bar", "parameter.2475805061.value"),
```

for additional testing to make sure the parameters were reset.